Gems/Atom/Tools/MaterialCanvas/Code/Source/Window/ViewportSettingsInspector/ViewportSettingsInspector.cpp

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Feature/Utils/LightingPreset.h>
+#include <Atom/Feature/Utils/ModelPreset.h>
+#include <Atom/RPI.Edit/Common/AssetUtils.h>
+#include <Atom/RPI.Reflect/System/AnyAsset.h>
+#include <AtomToolsFramework/AssetSelection/AssetSelectionGrid.h>
+#include <AtomToolsFramework/Inspector/InspectorPropertyGroupWidget.h>
+#include <AtomToolsFramework/Util/Util.h>
+#include <AzCore/Utils/Utils.h>
+#include <AzFramework/Application/Application.h>
+#include <Viewport/MaterialCanvasViewportSettingsRequestBus.h>
+#include <Window/ViewportSettingsInspector/ViewportSettingsInspector.h>
+
+#include <QAction>
+#include <QApplication>
+#include <QHBoxLayout>
+#include <QPushButton>
+#include <QToolButton>
+#include <QVBoxLayout>
+
+namespace MaterialCanvas
+{
+    ViewportSettingsInspector::ViewportSettingsInspector(const AZ::Crc32& toolId, QWidget* parent)
```

Comment:
Yes. It can be factored out. The lighting presets are being used in other places like previews, thumbnails, the animation editor, atom sample viewer, and will also be needed for the pass editor. I have suggested making a dedicated lighting preset editor instead of embedding it in the tools since there are so many other consumers now.